app/steps/ui/deceased/alias/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const ValidationStep = require('app/core/steps/ValidationStep');
+const DeceasedWrapper = require('app/wrappers/Deceased');
+const FormatName = require('app/utils/FormatName');
+
+class DeceasedAlias extends ValidationStep {
+
+    static getUrl() {
+        return '/deceased-alias';
+    }
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'alias', value: this.content.optionYes, choice: 'assetsInOtherNames'},
+            ]
+        };
+        return nextStepOptions;
+    }
+
+    getContextData(req) {
+        const ctx = super.getContextData(req);
+        const formdata = req.session.form;
+        ctx.deceasedName = FormatName.format(formdata.deceased);
+        return ctx;
+    }
```

Comment:
Are you not using the same replacement method you said David wants to use?

Reply:
agreed with David to leave template substitution as PA journey